src/components/OptionsList/BaseOptionsList.js

```diff
@@ -129,7 +189,12 @@ class BaseOptionsList extends Component {
     renderSectionHeader({section: {title, shouldShow}}) {
         if (title && shouldShow && !this.props.hideSectionHeaders) {
             return (
-                <View>
+
+                // Note: The `optionsListSectionHeader` style provides an explicit height to section headers.
+                // We do this so that we can reference the height in `getItemLayout` –
+                // we need to know the heights of all list items up-front in order to synchronously compute the layout of any given list item.
+                // So be aware that if you adjust the content of the section header (for example, change the font size), you may need to adjust this explicit height as well.
```

Comment:
> Where would I change the font size?

Nope, it would be on the next line – changing out `styles.textMicroBold` for something else.

> How do I do this?

You would change the `optionsListSectionHeader` style

Reply:
Ok still don't really love that we need to remember to update this, but not sure how to fix that without a more complicated solution like measuring the items dynamically (which doesn't make a ton of sense when things are fixed heights).